coreneuron/nrnoc/fadvance_core.cpp

```diff
@@ -289,8 +328,11 @@ void* nrn_fixed_step_lastpart(NrnThread* nth) {
 
         fixed_play_continuous(nth);
         nonvint(nth);
+        nrncore2nrn_send_values(nth);
         nrn_ba(nth, AFTER_SOLVE);
         nrn_ba(nth, BEFORE_STEP);
+    }else{
+        nrncore2nrn_send_values(nth);
```

Comment:
when `nt==NULL`, calling `nrncore2nrn_send_values` seems confusing at first sight.

Reply:
This is a case, not for nt==NULL but for nth->ncell  != 0 . I think I am imagining that someone may
want to plot a variable in an artificial cell.

Reply:
Ok understood !